model/task/task.go

```diff
@@ -669,6 +669,15 @@ func (t *Task) MarkEnd(finishTime time.Time, detail *apimodels.TaskEndDetail) er
 
 	t.TimeTaken = finishTime.Sub(t.StartTime)
 	t.Details = *detail
+
+	grip.Info(message.Fields{
+		"message":   "marking task finished",
+		"task_id":   t.Id,
+		"execution": t.Execution,
+		"project":   t.Project,
+		"status":    detail.Status,
+		"details":   t.Details,
```

Comment:
t.details should include status.

 also I doubt that t.details has good json tags.

Reply:
I don't think t.details is the same as status. There's a bunch of logic around this, and the fields mean different things. Since we're logging here, I think more data is better, and we should leave as is. t.details has fine JSON tags.

Reply:
nop.